apps/api/src/auth/auth.module.ts

```diff
@@ -50,6 +70,7 @@ import { GitlabStrategy } from '../config/oauth-strategy/gitlab/gitlab.strategy'
       inject: [GitlabOAuthStrategyFactory]
     }
   ],
-  controllers: [AuthController]
+  controllers: [AuthController],
+  exports: [AuthService]
```

Comment:
I doubted the same. It was a part of the original approach to prevent from duplicate instances being created.

https://stackoverflow.com/questions/77853711/nestjs-jwtservice-throws-error-secretorprivatekey-must-have-a-value-not-due-to

Reply:
![image](https://github.com/keyshade-xyz/keyshade/assets/83924254/f88e41d8-c3f6-4e57-9496-1c280c445d10)

Okay, but this is what the solution says. I don't think we should be exporting injectables as per the rules of DI.

Reply:
Sure! And do update the conflicts on your branch!